api/controllers/AuthController.js

```diff
@@ -21,7 +21,6 @@ module.exports = {
   },
 
   reddit: function(req, res) {
-    console.log(req);
```

Comment:
I think that's the only one -- I added it while trying to debug the mod authentication thing, but I forgot to remove it until now.